onebusaway-android/src/main/AndroidManifest.xml

```diff
@@ -37,6 +38,8 @@
     <uses-permission android:name="com.android.launcher.permission.INSTALL_SHORTCUT"/>
     <uses-permission android:name="${applicationId}.permission.TRIP_SERVICE"/>
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
+    <uses-permission android:name="com.google.android.gms.permission.ACTIVITY_RECOGNITION"/>
+    <uses-permission android:name="android.permission.REQUEST_IGNORE_BATTERY_OPTIMIZATIONS"/>
```

Comment:
Per my other comment I'll remove `REQUEST_IGNORE_BATTERY_OPTIMIZATIONS` after merge so we can easily revert to include it if needed after additional beta testing.